### ISSUE TITLE
Rotation doesn't work for videos

### ISSUE THREAD

**Issue by m-ocean-it** [SimpleMobileTools/Simple-Gallery#2188]
Can't rotate videos. Is it expected?

I long-press on the video, then select "Rotate" in the options (might be a different word since I use the Russian version), then choose "Rotate right" or "Rotate left". Nothing happens. Opening the video and rotating it with gestures doesn't work either.

I run GrapheneOS.

**Comment by tibbi** [comment]
yeah, videos are not supposed to be rotated like that, the menu button shouldnt be visible in those cases
